contribute.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "Standup",
+    "description": "Web app that logs daily standup updates",
+    "repository": {
+        "url": "https://github.com/mozilla/standup",
+        "license": "BSD-3-Clause",
+        "tests": "https://travis-ci.org/mozilla/standup/"
+    },
+    "participate": {
+        "home": "https://wiki.mozilla.org/Standup",
+        "docs": "https://github.com/mozilla/standup#readme",
```

Comment:
Perhaps for now "home" should be github and "docs" should be omitted until we setup RTD? I think of docs as the project docs.